README.md

```diff
@@ -45,7 +44,7 @@ $ cd ${YOUR_RIPPLE_SERIALIZE_DIRECTORY}
 $ git submodule update --init --recursive  # if you haven't already
```

Comment:
Both here, and on line 28, you can remove the `--recursive` parameter. With ripple-libpp, we needed it so that rippled would also get grabbed. But we don't need any of the submodules out of rippled.